UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/ModelRunner.cs

```diff
@@ -50,25 +50,25 @@ public class ModelRunner : IBatchedDecisionMaker
 
                 D.logEnabled = m_Verbose;
 
-                m_BarracudaModel = ModelLoader.Load(model.Value);
+                barracudaModel = ModelLoader.Load(model.Value);
                 var executionDevice = inferenceDevice == InferenceDevice.GPU
                     ? BarracudaWorkerFactory.Type.ComputePrecompiled
                     : BarracudaWorkerFactory.Type.CSharp;
-                m_Engine = BarracudaWorkerFactory.CreateWorker(executionDevice, m_BarracudaModel, m_Verbose);
+                m_Engine = BarracudaWorkerFactory.CreateWorker(executionDevice, barracudaModel, m_Verbose);
             }
             else
             {
-                m_BarracudaModel = null;
+                barracudaModel = null;
                 m_Engine = null;
             }
 
-            m_InferenceInputs = BarracudaModelParamLoader.GetInputTensors(m_BarracudaModel);
-            m_OutputNames = BarracudaModelParamLoader.GetOutputNames(m_BarracudaModel);
-            m_TensorGenerator = new TensorGenerator(brainParameters, seed, m_TensorAllocator, m_BarracudaModel);
-            m_TensorApplier = new TensorApplier(brainParameters, seed, m_TensorAllocator, m_BarracudaModel);
+            m_InferenceInputs = BarracudaModelParamLoader.GetInputTensors(barracudaModel);
+            m_OutputNames = BarracudaModelParamLoader.GetOutputNames(barracudaModel);
+            m_TensorGenerator = new TensorGenerator(brainParameters, seed, m_TensorAllocator, barracudaModel);
+            m_TensorApplier = new TensorApplier(brainParameters, seed, m_TensorAllocator, barracudaModel);
         }
 
-        private Dictionary<string, Tensor> PrepareBarracudaInputs(IEnumerable<TensorProxy> infInputs)
+        private Dictionary<string, Tensor> PrepareBarracudaInputs()
```

Comment:
Maybe make this static (and keep the IEnumerable argument) instead?

Reply:
makes sense.

Reply:
fixed